pandas/tests/test_strings.py

```diff
@@ -1072,28 +1072,44 @@ def test_extractall_single_group_with_quantifier(self):
         e = DataFrame(['ab', 'abc', 'd', 'cd'], i)
         tm.assert_frame_equal(r, e)
 
-    def test_extractall_no_matches(self):
-        s = Series(['a3', 'b3', 'd4c2'], name='series_name')
+    @pytest.mark.parametrize('data, names', [
+        ([], (None, )),
+        ([], ('i1', )),
+        ([], (None, 'i2')),
+        ([], ('i1', 'i2')),
+        (['a3', 'b3', 'd4c2'], (None, )),
+        (['a3', 'b3', 'd4c2'], ('i1', 'i2')),
+        (['a3', 'b3', 'd4c2'], (None, 'i2')),
+        (['a3', 'b3', 'd4c2'], ('i1', 'i2')),
+    ])
+    def test_extractall_no_matches(self, data, names):
```

Comment:
By test comments, @jreback meant a comment with the issue number, so like `# GH-19075` just below the `def test_`